### DISCUSSION TITLE
Add `related` and `relatedtype` fields to BibLaTeX exports

### DISCUSSION THREAD

**Discussion by alexreg** [retorquere/zotero-better-bibtex#2768] [Q&A] [answered]
What is the correct way to add the `related` and similar fields to biblatex exports? See section 3.5 of the biblatex manual. For example, output like the following.

```bibtex
@Book{key1,
  ...
  related     = {key2},
  relatedtype = {reprintof},
  ...
}
@Book{key2,
  ...
}
```

It looks like the following content in the `Extra` field of a Zotero entry would work, but I'm not sure it's the ideal solution.

```
biblatex.related: key2
biblatex.relatedtype: reprintof
```

**Comment by retorquere** [comment, marked as answer]
That is indeed the solution. I don't know of any Zotero information that could be used to generate these otherwise; Zotero knows generic relations but they are untyped.

**Reply by alexreg** [reply]
Thanks for the authoritative response. Yes, I'd come to the same conclusion about Zotero's concept of relations.
